addon/src/converter/fixture-converter.js

```diff
@@ -96,11 +96,13 @@ export default class FixtureConverter {
     }
   }
 
-  transformRelationshipKey(relationship) {
-    let { parentModelName, parentType } = relationship,
-      type =
+  transformRelationshipKey(relationship, parentModelName) {
+    const type =
+        /* parentModelName and parentType are not present on relationship schema in ember-data 5+, but these are here for backwards compat */
+        relationship.parentModelName ||
+        relationship.parentType?.modelName ||
+        /**/
         parentModelName ||
-        (parentType && parentType.modelName) ||
         relationship.type,
       transformFn = this.getTransformKeyFunction(type, 'Relationship');
```

Comment:
As the comment mentions. This was the one non-test change I had to make in supporting ember-data 5.3. Logic should be exactly the same for ED 3.28 users, and ED 5.x users will get the parentModelName from the args (which should be identical to what was being used before)